### PR TITLE
 [BD-29] [TNL-7268] Add high priority tests - conflicts resolved

### PR DESCRIPTION
**Pull request by davidjoy**

### CODE REVIEW

File: src/courseware/CoursewareContainer.test.jsx [hunk 1]
```diff
@@ -100,7 +99,7 @@ describe('CoursewareContainer', () => {
 
       expect(sequenceNavButtons[0]).toHaveTextContent('Previous');
       // Prove this button is rendering an SVG book icon, meaning it's a unit.
```
Comment (stvstnfrd) [inline src/courseware/CoursewareContainer.test.jsx]: Just to confirm, this is still checking for the book icon, not a "tasks" icon?

**Review by stvstnfrd — APPROVED** [review]

File: src/courseware/CoursewareContainer.test.jsx (hunk 1 above)
Comment (davidjoy) [reply src/courseware/CoursewareContainer.test.jsx]: Oops - forgot to update the comment.

**Comment by davidjoy** [comment]
@Agrendalath - conflicts are resolved here, and everything looks good!  I'm going to merge it in from here.  I mashed the committers a bit with a bit of sloppy git-fu, but since the commits are mostly preserved with authors, I think I'll just go ahead. :)

**Comment by Agrendalath** [comment]
@davidjoy, awesome, thank you for handling this and resolving all conflicts!:)

**Pull request merged**